README.adoc

```diff
@@ -552,6 +552,32 @@ validates_length_of :email, maximum: 100
 validates :email, presence: true, length: { maximum: 100 }
 ----
 
+=== Custom Validation Methods
+
+When naming custom validation methods, adhere to the simple rules:
+
+ - `validate :method_name` reads like a natural statement
+ - the method name explains what it checks
+ - the method is recognizable as a validation method by its name, not a predicate method
+
+[source,ruby]
+----
+# good
+validate :expiration_date_cannot_be_in_the_past
+validate :discount_cannot_be_greater_than_total_value
+validate :ensure_same_topic
```

Comment:
This last one doesn't read as naturally to me... maybe `validate :same_topic_is_chosen`?

Reply:
Maybe `ensure_same_topic_is_chosen`, WDYT? Without `ensure` it really looks like a predicate.

Reply:
My 'test' was to read it like a set of business rules and prefix each with "we":

"We validate expiration date cannot be in the past"
"We validate discount code cannot be total value"
"We validate ensure same topic" <- doesn't seem to flow

Reply:
We have several good sections with different naming patterns. The two in this section read well with "we", as they both are taken from the same source. The third one is from a different source.

Do you feel that it's necessary to have more good sections or more diversity in the primary good section?